Source Packages/io/rerum/crud/TinyOverwrite.java

```diff
@@ -144,6 +144,35 @@ protected void doPut(HttpServletRequest request, HttpServletResponse response)
             Logger.getLogger(TinyOverwrite.class.getName()).log(Level.SEVERE, null, ex);
         }
     }
+    
+    /**
+     * Handles the HTTP <code>OPTIONS</code> preflight method.
+     * This should be a configurable option.  Turning this on means you
+     * intend for this version of Tiny Things to work like an open API.  
+     *
+     * @param request servlet request
+     * @param response servlet response
+     * @throws ServletException if a servlet-specific error occurs
+     * @throws IOException if an I/O error occurs
+     */
+    @Override
+    protected void doOptions(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        try {
+            TinyTokenManager manager = new TinyTokenManager();
+            String openAPI = manager.getAPISetting();
+            if(openAPI.equals("true")){
```

Comment:
This is a good check. I can imagine others finding this to set up permission for just their other apps very easily.